### PR TITLE
Renames mixed gamemodes to have descriptive names

### PR DESCRIPTION
**Pull request by Yoshax**
Gives the mixed gamemodes proper names so the game mode can be derived from the name more easily, instead of having to guess or already know.

### CODE REVIEW

**Comment by nachomeep** [comment]
intrigue is ninjas not wizards dnm failure close pr ban yos

**Comment by Yoshax** [comment]
I don't know what you're talking about :)

**Comment by Kelenius** [comment]
Either make all of them use & or make all of them use "and".

**Comment by Yoshax** [comment]
Thought I had, my bad.

File: code/game/gamemodes/mixed/traitorling.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /datum/game_mode/traitorling
-	name = "traitorling"
+	name = "Traitors & Changelings"
```
Comment (Iamgoofball) [inline code/game/gamemodes/mixed/traitorling.dm]: traitorling was descriptive enough

Comment (Kelenius) [inline code/game/gamemodes/mixed/traitorling.dm]: No. Someone was getting confused regularly.

Comment (Iamgoofball) [inline code/game/gamemodes/mixed/traitorling.dm]: Seriously?

Comment (Kelenius) [inline code/game/gamemodes/mixed/traitorling.dm]: I can't say it happened often but it happened.

Comment (Yoshax) [inline code/game/gamemodes/mixed/traitorling.dm]: There's no reason not to make them entirely clear.

**Pull request merged**